checkIn/templates/admin/warnings.html

```diff
@@ -0,0 +1,75 @@
+{% extends "layout.html" %}
+{% block data %}
+<div class="container">
+	{% if error %}
+        <div class="row">
+            <div class="alert alert-danger text-center">
+                <strong>{{ error }}</strong>
+            </div>
+        </div>
+    {% endif %}
+	<div class="row">
+	    <div class="panel panel-default">
+	        <div class="panel-heading">
+	            <a href="/admin/lookup?sid={{ warnee.sid }}" class="btn btn-primary"><i class="glyphicon glyphicon-chevron-left"></i></a>
+	        </div>
+	        <ul class="list-group">
+	            <li class="list-group-item">
+		            <h3>Warn {{ warnee.name }}</h3>
+		            <form action="/admin/warnings/{{ warnee.sid }}" method="post">
+						<label for="reasonInput">Reason</label>
+		                <select name="category" id="category">
+							<option value=""{% if category != "Other" %} selected{% endif %} disabled hidden>Please select a category</option>
+			                <option value="Brought Prohibited Items">Brought Prohibited Items</option>
+			                <option value="Disorderly Conduct">Disorderly Conduct</option>
```

Comment:
* Safety glasses violation
* Improper tool use
* Food or drink in lab
* User agreement violation
* Blocking path of egress
* Damaging equipment/supplies
* Littering (or some nicer way to word leaving a mess)
* Improper attire